sqlmesh/core/context.py

```diff
@@ -800,21 +841,26 @@ def plan(
         model_selector = Selector(
             self.state_reader,
             self._models,
-            {
```

Comment:
why did you change this? multiple projects can have different defaults. so in a multi repo setup, you can have different default catalogs